tests/js/testCodingGameService.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-env jasmine */
```

Comment:
Does this need to be `/*` instead of `//`? (Just remembered that @cosimoc was nitpicking on this style of comments.)